packages/common/utils/passwordGenerator.js

```diff
@@ -0,0 +1,36 @@
+const DEFAULT_OPTIONS = {
+  symbols: {
+    digits: '1234567890',
+    specials: '!@#$%^&*()+-{}<>_:=',
+    lowerCase: 'abcdefghijklmnopqrstuvwxyz',
+    upperCase: 'ABCDEFGHIJKLMNOPQRSTUVWXYZ',
+  },
+  digits: true,
+  specials: true,
+  lowerCase: true,
+  upperCase: true,
+};
+const DEFAULT_LENGTH = 12;
+const getCrypto = () => window.crypto || window.msCrypto; // for IE 11
+const buildCharsSet = options => {
+  let validChats = '';
+  validChats += options.upperCase ? options.symbols.upperCase : '';
+  validChats += options.lowerCase ? options.symbols.lowerCase : '';
+  validChats += options.digits ? options.symbols.digits : '';
+  validChats += options.specials ? options.symbols.specials : '';
+  return validChats;
```

Comment:
// nitpick
I think it need to create an empty string before `return` for code readability

Reply:
We have the empty string at line 16, is it?

Reply:
No, I talk about 21 string.

Reply:
No, I talk about 21 string.